### PR TITLE
Upload the gcp yml files

### PR DESCRIPTION
**Pull request by XuyuFu**
I upload the .yml file of for gcp compute engine and resource manager, please check it. Thanks

### CODE REVIEW

File: Google_Compute_Instance.yml [hunk 1]
```diff
@@ -0,0 +1,888 @@
...
+    - contextPath: Gcp.Instance.User
+      description: '  If errors are generated during processing of the operation,
+        this field will be populated.'
+      type: string
+    - contextPath: Gcp.Instance.Error
+      description: '  If errors are generated during processing of the operation,
+        this field will be populated.'
+      type: string
+    description: Delete a certain instance
+  dockerimage: fuxuyu/demisto-gcp:4
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Create docker image for Demisto, upload it to our DockerHub repo and uploaded the docker image files to our git content repo

File: Google_Compute_Instance.yml [hunk 2]
```diff
@@ -0,0 +1,888 @@
...
+  name: client_id
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: client_x509_cert_url
+  name: client_x509_cert_url
+  defaultvalue: ""
+  type: 12
+  required: true
+script:
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Use our integration docstring template, for example `"""Imports"""` is missing here

File: Google_Compute_Instance.yml [hunk 3]
```diff
@@ -0,0 +1,888 @@
...
+    data['client_id'] = demisto.params()['client_id']
+    data['auth_uri'] = "https://accounts.google.com/o/oauth2/auth"
+    data['token_uri'] = "https://oauth2.googleapis.com/token"
+    data['auth_provider_x509_cert_url'] = "https://www.googleapis.com/oauth2/v1/certs"
+    data[
+        'client_x509_cert_url'] = demisto.params()['client_x509_cert_url']
+
+    def create_gcompute_instance(args):
+        credentials = service_account.Credentials.from_service_account_info(data)
+        compute = googleapiclient.discovery.build('compute', 'v1', credentials=credentials);
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: remove `;`

File: Google_Compute_Instance.yml [hunk 4]
```diff
@@ -0,0 +1,888 @@
...
+
+        image_type = 'ubuntu-1710-artful-v20180315'
+        project_type = 'ubuntu-os-cloud'
+        description = "N/A"
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: `"` -> `'`

File: Google_Compute_Instance.yml [hunk 5]
```diff
@@ -0,0 +1,888 @@
...
+        tags = None
+        networkInterfaces = []
+        networkInterfaces.append({
+                'network': 'global/networks/default',
+                'accessConfigs': [
+                    {'type': 'ONE_TO_ONE_NAT', 'name': 'External NAT'}
+                ]
+            })
+
+        project = args.get( 'projectId' )
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Remove white spaces inside the brackets

File: Google_Compute_Instance.yml [hunk 6]
```diff
@@ -0,0 +1,888 @@
...
+        zone = args.get( 'zone' )
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: You can just iterate all `args` instead of it doing it one by one

File: Google_Compute_Instance.yml [hunk 7]
```diff
@@ -0,0 +1,888 @@
...
+            tags = args.get('tags')
+        if args.get('networkInterfaces') is not None:
+            networkInterfaces = args.get('networkInterfaces')
+
+        image_response = compute.images().get( project=project_type, image=image_type ).execute()
+        source_disk_image = image_response['selfLink']
+
+        # Configure the machine
+        machine_type = "zones/%s/machineTypes/n1-standard-1" % zone
+        # startup_script = open(os.path.join(os.path.dirname(__file__), 'startup-script.sh'), 'r').read()
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Remove comment

File: Google_Compute_Instance.yml [hunk 8]
```diff
@@ -0,0 +1,888 @@
...
+        image_url = "http://storage.googleapis.com/gce-demo-input/photo.jpg"
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: What is this?

File: Google_Compute_Instance.yml [hunk 9]
```diff
@@ -0,0 +1,888 @@
...
+        if args.get('image_type') is not None:
+            image_type = args.get('image_type')
+        if args.get('project_type') is not None:
+            project_type = args.get('project_type')
+        if args.get('description') is not None:
+            description = args.get('description')
+        if args.get('canIpForward') is not None:
+            canIpForward = args.get('canIpForward')
+        if args.get('deletionProtection') is not None:
+            deletionProtection = args.get('deletionProtection')
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Use Python coding convention, should be `deletion_protection`

File: Google_Compute_Instance.yml [hunk 10]
```diff
@@ -0,0 +1,888 @@
...
+            'disks': [
+                {
+                    'boot': True,
+                    'autoDelete': True,
+                    'initializeParams': {
+                        'sourceImage': source_disk_image,
+                    }
+                }
+            ],
+
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Remove blank lines

File: Google_Compute_Instance.yml [hunk 11]
```diff
@@ -0,0 +1,888 @@
...
+        md = tableToMarkdown('Created a new Google Compute Instance',humanread)
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': operation,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable':md,
+            'EntryContext': {
+                'Gcp.instance': response
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Set unique field, i.e. `ProjectId`

File: Google_Compute_Instance.yml [hunk 12]
```diff
@@ -0,0 +1,888 @@
...
+        zone = args.get( 'zone' )
+        name = args.get( 'instanceName' )
+
+        operation = compute.instances().stop(project=project, zone=zone, instance=name).execute()
+
+        response = {}
+        response['ProjectId'] = operation['id']
+        response['InstanceName'] = operation['name']
+        response['Zone'] = operation['zone']
+        # response['Instance status'] = operation['status']
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: remove notes

File: Google_Compute_Instance.yml [hunk 13]
```diff
@@ -0,0 +1,888 @@
...
+        humanread= {
+            'Instance Name': operation['name'],
+            'Instance Zone':operation['zone'],
+            'Instance Status':operation['status'],
+            'Command initiated by':operation['user'],
+            'Instance errors':"N/A"
+        }
+
+        erroroupt =[]
+        if 'error' in operation:
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: If there's an error, you should use `return_error`

File: Google_Compute_Instance.yml [hunk 14]
```diff
@@ -0,0 +1,888 @@
...
+            'Instance Labels':'N/A' if 'labels' not in operation else operation['labels'],
+            'Instance Can IP Forward':operation['canIpForward'],
+            'Instance Deletion Protection':operation['deletionProtection'],
+        }
+
+
+        md = tableToMarkdown('Details of the requested Google Compute Instance', humanread)
+        md1 = tableToMarkdown('Network Interfaces', humanread_networks)
+        md2 = tableToMarkdown('Disks', humanread_disk)
+        # entry = {
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Remove comments

File: Google_Compute_Instance.yml [hunk 15]
```diff
@@ -0,0 +1,888 @@
...
+        #     'HumanReadable':md,
+        #     'HumanReadable':md1,
+        #     'HumanReadable':md2,
+        #     'EntryContext': {
+        #         'Gcp.Instance': response
+        #     }
+        # }
+
+        entry = []
+        entry.append( {
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: There is no need in so many entries. Just concat all the tables in `md`

File: Google_Compute_Instance.yml [hunk 16]
```diff
@@ -0,0 +1,888 @@
...
+        get_gcompute_instance( demisto.args() )
+
+    if demisto.command() == 'gcompute-delete-instance':
+        delete_gcompute_instance( demisto.args() )
+
+  type: python
+  commands:
+  - name: gcompute-create-instance
+    arguments:
+    - name: projectId
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Should be `project_id`

File: Google_Compute_Instance.yml [hunk 17]
```diff
@@ -0,0 +1,888 @@
...
+      - europe-west4-a/b/c
+      - northamerica-northeast1-a/b/c
+      - southamerica-east1-a/b/c
+      - us-central1-a/b/c/f
+      - us-east1-b/c/d
+      - us-east4-a/b/c
+      - us-west1-a/b/c
+      - us-west2-a/b/c
+      description: The name of the zone for this request. For example, us-east1-b
+      defaultValue: Choose from auto-populated values. Use one of the a, b, c, d or
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: How is this a default value?

File: Google_Compute_Instance.yml [hunk 18]
```diff
@@ -0,0 +1,888 @@
...
+        or creationTimestamp desc is supported.
+    - name: maxResults
+      description: 'the max number of instances listed. Acceptable values are 0 to
+        500, inclusive. (Default: 500)'
+    - name: filter
+      description: the filter applied to the instances-specify the field name, a comparison
+        operator, and the value that you want to use for filtering
+    outputs:
+    - contextPath: Gcp.Instance.Status
+      description: ' The status of the instance. '
```
Comment (Itay4) [inline Google_Compute_Instance.yml]: Remove white spaces

**Comment by yaakovi** [comment]
@XuyuFu thanks for submitting this. I did a rebase on the branch as we recently did a rebase on our main repo. The branch is now in sync with our main repo.

**Comment by Itay4** [comment]
GRM already implemented and GCI will be implemented as a fix for https://github.com/demisto/etc/issues/14042

**Pull request closed without merging**